src/components/icons/staking/RockXGlyphIcon.tsx

```diff
@@ -5,7 +5,7 @@ import { createIconBase } from "../icon-base"
 export const RockXGlyphIcon = createIconBase({
   displayName: "RockXGlyphIcon",
   viewBox: "0 0 32 32",
-  className: "h-auto w-[32px]",
+  className: "size-[1em] bg-black p-2 dark:bg-background-highlight",
```

Comment:
Thanks @rafagomes! I see where you're going here, but wouldn't recommend this approach here tbh. Ideally we avoid declaring specific colors here as much as possible, unless we really want to force them. In this case, we still aren't applying the proper brand colors, so we want this to be responsive to grayscale depending on light or dark mode. ie, we want it to invert.

Playing locally, I updated this icon to the following:

```tsx
import { commonIconDefaultAttrs } from "@/components/icons/utils"

import { createIconBase } from "../icon-base"

export const RockXGlyphIcon = createIconBase({
  displayName: "RockXGlyphIcon",
  viewBox: "0 0 32 32",
  className: "size-[1em]",
  ...commonIconDefaultAttrs,
  children: (
    <>
      <path
        d="M19.0337 19.0337C20.7092 17.3583 20.7092 14.6417 19.0337 12.9663L7.32408 1.25661C5.64859 -0.41888 2.93207 -0.418859 1.25661 1.25661C-0.418859 2.93207 -0.41888 5.64859 1.25661 7.32408L9.93253 16L4.09187 21.8406C2.41641 23.5161 2.41639 26.2326 4.09187 27.9081C5.76736 29.5836 8.48388 29.5836 10.1593 27.9081L19.0337 19.0337Z"
        opacity="0.6"
      />

      <path
        d="M12.9675 12.9662C11.2921 14.6417 11.2921 17.3582 12.9675 19.0337L24.6772 30.7433C26.3527 32.4188 29.0692 32.4188 30.7447 30.7433C32.4201 29.0679 32.4202 26.3513 30.7447 24.6759L22.0688 15.9999L27.9094 10.1593C29.5849 8.48382 29.5849 5.7673 27.9094 4.09181C26.2339 2.41633 23.5174 2.41635 21.8419 4.09181L12.9675 12.9662Z"
        opacity="0.8"
      />
    </>
  ),
})
```

1. I reverted the className back to just `className: "size-[1em]",`
2. Realized the third path was an unnecessary duplicate, so I removed it
3. Removed the `fill` attribute from the paths, since `fill="currentColor"` is already applied to everything via `...commonIconDefaultAttrs`
4. Added `opacity` attributes (to specific paths) instead for this icon specifically, so give it an appearance of the two "arrows" intersecting with each other.

With those changes, you can see how this becomes nicely responsive in light and dark modes, similar to the other logos:

<img width="822" alt="image" src="https://github.com/user-attachments/assets/c5a20975-47e0-44f2-8bbb-ca27a75fa342">

<img width="848" alt="image" src="https://github.com/user-attachments/assets/c9cf4ef4-b5d0-48e2-b932-fc31a34e1f2b">